docs/detections/rules-ui-create.asciidoc

```diff
@@ -352,9 +352,14 @@ NOTE: If you select this option, you can add
 Additionally, all future exceptions added to the Elastic Endpoint Security rule
 also affect this rule.
 +
+
 .. *Building block* (optional): Select to create a building-block rule. By
 default, alerts generated from a building-block rule are not displayed in the
 UI. See <<building-block-rule>> for more information.
+.. *Indicator prefix override*: Define the data structure path of indicator data inside indicator documents. When the indicator match rule executes, it queries specified indicator indices and references this setting to locate fields with indicator data. This data is used to enrich indicator match alerts with metadata about matched threat indicators. The default value for this field is `threat.indicator`.
```

Comment:
maybe we can say something like `Define the location of enrichment data within the structure of the indicator document`. Because there is all kinds of data in the indicators that are outside this path as well, but only what's within the threat_indicator_path will be used as the enrichment